geometry/Goal.cpp

```diff
@@ -234,16 +232,16 @@ bool Goal::ConvertLineToPoly()
           point1 = tmp +  diff * 0.95;
           point2 = tmp +  diff * 0.05;
 
-          _crossing->SetPoint1(point1);
-          _crossing->SetPoint2(point2);
+          _crossing.SetPoint1(point1);
+          _crossing.SetPoint2(point2);
      }else{
-          _crossing->SetPoint1(_poly[0]);
+          _crossing.SetPoint1(_poly[0]);
           Line tmp_line(_poly[_poly.size()/2], _poly[(_poly.size()/2)+1], 0);
-          _crossing->SetPoint2(tmp_line.GetCentre());
+          _crossing.SetPoint2(tmp_line.GetCentre());
      }
 
 
-//     std::cout << "Crossing goal: " << _crossing->GetUniqueID() << _crossing->toString() << std::endl;
```

Comment:
Can be deleted. Blame me for the comment.

Reply:
removed ;)